src/crypt_special.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Created by tobias on 01.06.21.
+//
+
+#include <crypt_special.h>
+#include <cstring>
+
+static char MIN_PRINTABLE = 0x20;
+static char MAX_PRINTABLE = 0x7e;
+
+
+static char S1[] = {
+        0x22, 0x15, 0x11, 0x0b,
+        0x42, 0x43, 0x26, 0x42,
+        0x24, 0x4c, 0x33, 0x10,
+        0x11, 0xa, 0x56, 0x23
+};
+
```

Comment:
Hey @sirtoobii,
thany you very much for contributing!
Could you please add a small comment line describing the functionality and need of this Special Hash function?

Thanks a lot!